master/contrib/bzr_buildbot.py

```diff
@@ -96,6 +96,7 @@
 """
 
 try:
+    import klog
```

Comment:
I think existing this import is necessary for poller, so it shouldn't be in `try` block